app/start/global.php

```diff
@@ -57,9 +57,13 @@
 });
 
 App::missing(function ($exception) {
-    return Response::view('errors.404', [
-        'pageTitle' => 'That page went missing!'
-    ], 404);
+    if (Request::is('dashboard*') or Request::is('api*')) {
+        return Response::view('errors.404', [
+            'pageTitle' => 'That page went missing!'
```

Comment:
I've got a feeling we got that wrong in php-cs-fixer... We messed up the multiline function stuff before...